utam-compiler/src/test/java/utam/compiler/translator/InterfaceSerializerTests.java

```diff
@@ -0,0 +1,105 @@
+package utam.compiler.translator;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.not;
+import static utam.compiler.grammar.TestUtilities.getJsonStringDeserializer;
+
+import org.testng.annotations.Test;
+import utam.core.declarative.representation.PageObjectInterface;
+
+public class InterfaceSerializerTests {
+
+  private static String getImplementationCode(String json) {
+    PageObjectInterface pageObject = getJsonStringDeserializer(json).getObject().getInterface();
+    return new InterfaceSerializer(pageObject).toString();
+  }
+
+  @Test
+  public void testToString() {
+    String json =
```

Comment:
This isn't the minimal test for the change and the test name isn't meaningful. If this fails, how would someone know what was supposed to be happening?

Reply:
The extant test will be blatantly obvious what the issue is. We can undertake a test refactoring later if we really feel strongly about this.